website/client/src/components/inventory/stable/index.vue

```diff
@@ -778,6 +779,16 @@ export default {
 
       return animals;
     },
+    getTextOwnedAnimals (petGroup, pets) {
+      return ` (${this.getCountOwnedAnimals(petGroup, 'pet')} ${petGroup.label} ${pets})`;
+    },
+    getCountOwnedAnimals (animalGroup, type) {
```

Comment:
Instead of introducing a new computed property can we reuse this new one inside `countOwnedAnimals` so the code isn't duplicated? Thanks

Reply:
Ok, incorporated changes!

Btw, is the name `getCountOwnedAnimals` ok, or should it better be `getCountOwnedPets`?